web/docker/Dockerfile

```diff
@@ -80,17 +57,36 @@ RUN set -x && apt-get update -qq \
     # To switch user and exec command.
     gosu
 
-COPY --from=builder /codechecker/web/build/CodeChecker /codechecker/
+RUN if [ "$INSTALL_AUTH" = "yes" ] ; then \
+      apt-get install -qqy --no-install-recommends \
+        libldap2-dev \
+        libsasl2-dev \
+        libssl-dev; \
```

Comment:
Shouldn't we install `libpq-dev` for psycopg2?
https://github.com/Ericsson/codechecker/blame/4203d51478c73c48d34eb7edb5a6190ddf77be3c/docs/web/postgresql_setup.md#L23-L24